### ISSUE TITLE
3rd-Party Auth Client Credentials Grant

### ISSUE THREAD

**Issue by hmiguel** [cloudendpoints/endpoints-python#122]
Hey, 

I'm trying to request my api with a [Client Credentials Grant](https://tools.ietf.org/html/rfc6749#section-4.4) access token, but it seems to fail when tries to get `user_info.email` (I guess) which is None, because Client Credentials Grant doesn't have user info.

https://github.com/cloudendpoints/endpoints-python/blob/4766d87b4d66379d350848754a1d1670aae12aba/endpoints/users_id_token.py#L119

Api definition:

`@endpoints.api(name='todo',version='v1', audiences={'auth0': ["http://api.todo.com"]},
        issuers={'auth0': endpoints.Issuer("https://todo.eu.auth0.com/","https://todo.eu.auth0.com/.well-known/jwks.json")})`

Any idea how to do this?

In other words, are endpoints standard framework v2 prepared to deal with access tokens?

**Comment by inklesspen** [comment]
The Endpoints Framework does not support access tokens from authentication providers other than Google. Only id tokens (JWTs) are supported from auth0.
